client/src/app/site/site.component.html

```diff
@@ -46,15 +46,14 @@
         <!-- navigation -->
         <mat-nav-list class='main-nav'>
             <span *ngFor="let entry of mainMenuService.entries">
-                <a [@navItemAnim]  *osPerms="entry.permission" mat-list-item (click)='toggleSideNav()'
+                <a [@navItemAnim] *osPerms="entry.permission" mat-list-item (click)='toggleSideNav()'
                     [routerLink]='entry.route' routerLinkActive='active' [routerLinkActiveOptions]="{exact: true}">
-                    <fa-icon [icon]='entry.icon'></fa-icon>
-                    {{ entry.displayName | translate}}
+                    <mat-icon>{{entry.icon}}</mat-icon>{{ entry.displayName | translate}}
                 </a>
             </span>
             <mat-divider></mat-divider>
             <a [@navItemAnim] *osPerms="'core.can_see_projector'" mat-list-item routerLink='/projector' routerLinkActive='active' (click)='toggleSideNav()'>
-                <fa-icon icon='video'></fa-icon>
+                <mat-icon>videocam</mat-icon>
```

Comment:
`cast` does not really belong to my old-fashioned world (I would not recognize this). Maybe others feel the same?

Reply:
:man_shrugging: 
video-camera-icons are also biased, since actually, we are not recording anything (yet).
In my opinion, the cast icon comes closer to projection than the video-camera icon.

How are the others feeling about this?
@emanuelschuetze @jsaalfeld @FinnStutzenstein

Reply:
Lets keep the camera icon.. If we change to material icons, we should keep the logos similar.

Reply:
@emanuelschuetze agreed that the cast-icon might be too biased by chromecast. We would stick for the projector for the time being. In the end, we have to see it how it works in the list-view once it has a "projectable" indicator.

Stick with videocam for now.